site/pages/api/get-all-disruptions.api.ts

```diff
@@ -6,7 +6,12 @@ import { Dayjs } from "dayjs";
 import { NextApiRequest, NextApiResponse } from "next";
 import { VEHICLE_MODES } from "../../constants";
 import { getDisruptionsDataFromDynamo } from "../../data/dynamo";
-import { getDisplayByValue, mapValidityPeriods, reduceStringWithEllipsis } from "../../utils";
+import {
+    filterDisruptionsForOperatorUser,
```

Comment:
About AC3 Be able to view disruptions created by other LTA user roles (admin, publisher, staff) that have status: Open, Closed, Closing. 
I can see drafts made by LTA user roles